src/Microsoft.AspNet.Razor/RazorEditorParser.cs

```diff
@@ -12,45 +12,70 @@
 namespace Microsoft.AspNet.Razor
 {
     /// <summary>
-    /// Parser used by editors to avoid reparsing the entire document on each text change
+    /// Parser used by editors to avoid reparsing the entire document on each text change.
     /// </summary>
     /// <remarks>
+    /// <para>
     /// This parser is designed to allow editors to avoid having to worry about incremental parsing.
-    /// The CheckForStructureChanges method can be called with every change made by a user in an editor and
-    /// the parser will provide a result indicating if it was able to incrementally reparse the document.
-    ///
+    /// The <see cref="CheckForStructureChanges"/> method can be called with every change made by a user in an editor
+    /// and the parser will provide a result indicating if it was able to incrementally reparse the document.
```

Comment:
`indicating if it was able to incrementally reparse the document.` => `indicating if it was able to apply the change.`